tests/integration/test_verify.py

```diff
@@ -16,6 +17,24 @@ def test_verify_valid_sources(source_dir):
     assert exit_code == 0
 
 
+@pytest.mark.parametrize('source_dir,output', [
+    ("tests/test_files/yaml_source_dir/valid_yamls_with_multiple_crds",
+     "tests/bundle.yaml"),
```

Comment:
Maybe safer to put the output file in `/tmp`, just in case the test is aborted in the middle and the output file doesn't get deleted?

Reply:
+1, you can use pytest's `tmpdir` fixture or `tempfile` module